eos-update-flatpak-repos

```diff
@@ -709,12 +615,5 @@ if __name__ == '__main__':
     # ensure default remotes exist, such as eos-sdk and flathub
     _add_flatpak_repos()
 
-    # remove unused remotes, such as eos-external-apps and gnome-apps
-    _remove_remotes()
-
-    # clear away legacy eos-external-apps repo and runtimes (T14442)
-    _remove_old_external_apps_repo()
-    _remove_runtimes()
```

Comment:
Nitpick: I’d just empty the lists which drive these functions, since we might want to remove other repos in future, and doing archaeology to resurrect the code for that would be a pain.